command/command.c

```diff
@@ -289,21 +295,71 @@ process_command_output (CommandApplet *command_applet, gchar *output)
     }
 }
 
+static void
+subprocess_callback (GObject *source, GAsyncResult *result, gpointer data)
+{
+    GSubprocess *subprocess;
+    CommandApplet *command_applet;
+    GInputStream *in;
+    gsize bytes_read;
+    gchar *output;
+    GError *error = NULL;
+
+    command_applet = (CommandApplet *) data;
+    subprocess = G_SUBPROCESS (command_applet->subprocess);
+
+    if (!g_subprocess_wait_finish (subprocess, result, &error))
+    {
+        g_warning ("Error: %s\n", error->message);
```

Comment:
Why do you add newline? g_warning, g_debug... does that for you.

Reply:
They have been deleted.